alignak/objects/config.py

```diff
@@ -1401,13 +1419,17 @@ def prepare_for_sending(self):
 
         :return: None
         """
+        logger.info('Prepare the configuration sending')
+
+        # Alert about using this method that is no more useful
+        warnings.warn("Using Config::prepare_for_sending, still useful?",
```

Comment:
I raised a warning to confirm that it is really not used during the test ... rather than raising an exception